zerver/views/streams.py

```diff
@@ -625,16 +625,22 @@ def send_messages_for_new_subscribers(
         notifications_stream = user_profile.realm.get_notifications_stream()
         if notifications_stream is not None:
             with override_language(notifications_stream.realm.default_language):
-                if len(created_streams) > 1:
-                    content = _("{user_name} created the following streams: {stream_str}.")
-                else:
-                    content = _("{user_name} created a new stream {stream_str}.")
                 topic = _("new streams")
 
-            content = content.format(
-                user_name=f"@_**{user_profile.full_name}|{user_profile.id}**",
-                stream_str=", ".join(f"#**{s.name}**" for s in created_streams),
-            )
+                user_name = f"@_**{user_profile.full_name}|{user_profile.id}**"
+
+                content = "\n\n".join(
+                    map(
+                        lambda stream: _(
+                            "{user_name} created a new stream #**{stream.name}** (*{stream.description}*)"
```

Comment:
This should probably end with a `).` for consistency with the other form being a sentence.